e2e/ganache/index.js

```diff
@@ -0,0 +1,41 @@
+const ganache = require('ganache-cli')
+const getPort = require('get-port')
+
+class Ganache {
+  async start (options) {
+    const port = options.port || await getPort()
+    this._server = ganache.server(options)
+    return new Promise((resolve, reject) => {
```

Comment:
Some of the comments I made in the `FixtureServer` apply here too:
* Should check for existing `this._server`, otherwise it would be shadowed and maybe never stopped.
* Should use `promisify` instead of the Promise constructor, both here and in `quit`
  Well, the case is less compelling here because there aren't any unhanded error scenarios. Still seems like a weird mix of paradigms though.